web/console/src/app/views/CardPage/index.tsx

```diff
@@ -48,7 +48,7 @@ const Card: React.FC = () => {
         }
     }
 
-    const mint = async () => {
+    const mint = async() => {
         const walletService = new WalletService(user);
         await walletService.mintNft(id);
```

Comment:
add try catch

Reply:
Try catch is in casper/velas/ ethers services, there are notifications about errors too